elements/pl-workspace/pl-workspace.py

```diff
@@ -12,3 +18,24 @@ def render(element_html, data):
         html = chevron.render(f, html_params).strip()
 
     return html
+
+
+def parse(element_html, data):
```

Comment:
This function seems to have a lot of checks that aren't needed. I think we could just do something like:
```
workspace_required_file_names = data['submitted_answers'].get('_workspace_required_file_names', [])
submitted_file_names = [f.get('name', '') for f in data['submitted_answers'].get('_files', [])]
missing_files = [n in workspace_required_file_names if n not in submitted_file_names]
if (missing_files):
    add_format_error(data, f'The following required files were missing: {", ".join(missing_files)}')
```

(see comment below about `_workspace_required_file_names`.

Reply:
Done. Originally I had more or less copied the `file-*` checks and errors. I thought that we'd get more consistent errors that way (e.g., `No submitted answer for _____`).

Reply:
I guess I was worried about the use of the phrase "submitted answer" there. For workspaces I think students might not be thinking about a "submitted answer" as such. But if we say "these files were missing" then hopefully it's clear that those files needed to be present?

Reply:
Sure, sounds good